internal/credential/credential.go

```diff
@@ -106,7 +106,7 @@ type Issuer interface {
 	//
 	// If Issue encounters an error, it returns no credentials and revokes
 	// any credentials issued before encountering the error.
```

Comment:
Can we update the doc to explain what the valid options are?

Reply:
Done